### PR TITLE
Dynamic pitch on Howler

### PR DESCRIPTION
**Pull request by Lizard-13**
Can we add it?, cocos doesn't support it but from what I can see cocos doesn't support lot of things :D

### CODE REVIEW

File: GDJS/Runtime/events-tools/soundtools.js [hunk 1]
```diff
@@ -87,6 +87,11 @@ gdjs.evtTools.sound.getSoundOnChannelPitch = function(runtimeScene, channel) {
     return sound ? sound.rate() : 1;
```
Comment (4ian) [inline GDJS/Runtime/events-tools/soundtools.js]: I think you may have forgotten to update this line to use `getRate`? :)

File: GDJS/Runtime/howler-sound-manager/howler-sound-manager.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ gdjs.HowlerSound = function(o) {
     this._paused = false;
     this._stopped = true;
     this._canBeDestroyed = false;
-    this._rate = o.rate || 1; //Read-only
+    this._rate = o.rate || 1;
```
Comment (4ian) [inline GDJS/Runtime/howler-sound-manager/howler-sound-manager.js]: 👍 Thanks for removing the comment

File: GDJS/Runtime/events-tools/soundtools.js (hunk 1 above)
Comment (Lizard-13) [reply GDJS/Runtime/events-tools/soundtools.js]: Absolutely!, fixed :)

**Comment by Lizard-13** [comment]
Fixed

**Pull request merged**